clients/admin-ui/src/features/system/form.ts

```diff
@@ -124,6 +124,7 @@ export const transformFormValuesToSystem = (formValues: FormValues): System => {
     fidesctl_meta: formValues.fidesctl_meta,
     organization_fides_key: formValues.organization_fides_key,
     dpa_progress: formValues.dpa_progress,
+    cookies: formValues.cookies,
```

Comment:
this is pretty much the change that made things work!

Reply:
ah okay thank you for pointing this out!